README.md

```diff
@@ -190,7 +190,13 @@ Each cluster that will be connected must have Submariner installed within it. Yo
    ```
    kubectl -n kube-system  rollout status deploy/tiller-deploy
    ```
+1. Grant  the appropriate security context for the service accounts (if required)
    
+   For installation based on platforms like Openshift, the service account user needs to be associated with the appropriate security context. This may not be required for Kubernetes based installations as by default security context constraints are not configured in Kubernetes.
```

Comment:
"Kubernetes based" might be better as "Kubernetes-based"

Reply:
Of course Openshift is "Kubernetes-based" :)  Perhaps what we want to say "pure Kubernetes installations".